monitor/extensions/referencedata-wizards/src/main/resources/org/datacleaner/monitor/server/wizard/dictionary/simple/SimpleDictionaryReferenceDataPage.html

```diff
@@ -0,0 +1,29 @@
+<div style="min-height: 200px; background-repeat: no-repeat; background-position: bottom right; ">
+    <form action="repository/demo/jobs/multiple_upload" method="post" enctype="multipart/form-data">
```

Comment:
Actually, this is a forgotten artifact. It is not used. I will remove it. Thanks for noticing it. ;-)